packages/@okta/courage-for-signin-widget.js

```diff
@@ -0,0 +1,11371 @@
+(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
Similar to what @mauriciocastillosilva-okta asked on slack. Should this be done within courage as opposed to doing it here in the widget? Feels like the consumer should not be  responsible for this . Additionally I am not sure if the current work of moving courage to okta-ui has any affect on this.

Reply:
why would think consumer should not be responsible for this?
yeah, it could be done in courage and I'm slightly leaning toward in signin widget as the first step. (didn't add complexity to Courage plus improve the signin widget a little bit)
we could move to courage or another okta-ui package after courage has been moved to okta-ui. 
- probably another office hour topic?

Reply:
This is great first step and maybe worth another office hour discussion. Only reason I feel this could belong in courage is
- Since this is in the open-source sign in widget repo and the courage framework is only internal to okta, all this bundling logic only makes sense to okta.
- Comparing it to how we have option to bundle  courage with `okta.js` and `OktaWithInputs ` or in the sign-in-widget we have  `cdnConfig` and `noJqueryConfig `, feels like this could be another option where we ship courage for sign in widget as another option.

> we could move to courage or another okta-ui package after courage has been moved to okta-ui.`. 

Yeah, if we do eventually want this to live in  okta-ui as a package or under courage as a package? hopefully this would make it easier to move later.

Reply:
Just my two cents -

The Widget is a consumer of Courage, so it should be responsible for any/all bundling logic around it. Just as any other library uses a component package, it too is responsible to filtering out what it needs and/or doesn't need.

If we move this logic within Courage, it **does** simplify the build steps + dependencies for the Widget. It could simply import `@okta/courage@4.4.0-beta.XYZ/dist/CourageForSigninWidget.js`, and be done.

That being said, I'm +1 for having an office hours discussion around this. Further, I'm curious what versioning strategy for "internal packages" can be. With both implementations, we're forced to manually update the courage version, while a flexible range might be a viable feature.

Reply:
I also think this logic to build a custom version of Courage for Widget consumption should live in the Widget repo. The folks maintaining the Widget will have a better idea of what the Widget requires from Courage than the folks maintaining Courage. My $0.02 🙂